index.html

```diff
@@ -0,0 +1,309 @@
+    <title>VNN-COMP - International Verification of Neural Networks Competition</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="header-content">
+                <h1 class="logo">VNN-COMP</h1>
+                <p class="tagline">International Verification of Neural Networks Competition</p>
+            </div>
+            <nav class="nav">
+                <ul class="nav-list">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#competitions">Competitions</a></li>
+                    <li><a href="#repositories">Repositories</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section id="hero" class="hero">
+            <div class="container">
+                <div class="hero-content">
+                    <h2 class="hero-title">Advancing Neural Network Verification</h2>
+                    <p class="hero-description">
+                        VNN-COMP is the premier international competition dedicated to evaluating and advancing 
+                        the state-of-the-art in neural network verification. Join researchers and practitioners 
+                        worldwide in pushing the boundaries of AI safety and reliability.
+                    </p>
+                    <div class="hero-buttons">
+                        <a href="#competitions" class="btn btn-primary">View Competitions</a>
+                        <a href="#repositories" class="btn btn-secondary">Browse Repositories</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="about" class="about">
+            <div class="container">
+                <h2 class="section-title">About VNN-COMP</h2>
+                <div class="about-content">
+                    <div class="about-text">
+                        <p>
+                            The International Verification of Neural Networks Competition (VNN-COMP) brings together 
+                            researchers, practitioners, and tool developers to evaluate and compare neural network 
+                            verification methods. Our mission is to advance the field by providing standardized 
+                            benchmarks, fostering collaboration, and promoting best practices in neural network safety.
+                        </p>
+                        <p>
+                            Since 2019, VNN-COMP has served as a central hub for the neural network verification 
```

Comment:
while the vnn workshop was held in 2019 and ideas like vnn-comp were discussed, the vnn-comp was launched in 2020